### DISCUSSION TITLE
Creating an account with first name and last name fields

### DISCUSSION THREAD

**Discussion by asimolmez** [serverpod/serverpod#1472] [Q&A] [answered]
I develop a create account screen and trying to save full name of user. Is it possible to indicate full name of user? i have checked `createAccountRequest` signature, and i see only `email`, `password` and `username`.
Thanks,

**Comment by Isakdl** [comment, marked as answer]
You can create your own table and store any data you like for the user!

**Reply by asimolmez** [reply]
Gotcha, thank you!
